app/views/pages/now.html.slim

```diff
@@ -1,7 +1,9 @@
 section
   #next-meeting
     h2 Next Meeting
-    p= formatted_date(@next_crb.time)
+    p
+      Food and networking: 5pm-6:30pm
+      Program: #{formatted_date(@next_crb.time)}
```

Comment:
The #{} around this is causing a syntax error. 
```
Expected attribute
  /app/app/views/pages/now.html.slim, Line 6, Column 17
    Program: #{formatted_date(@next_crb.time)}
               ^
```

You mentioned that you were not familiar with slim, but you should be able to validate your changes by following the steps in the https://github.com/columbusrb/columbusrb.com/blob/master/contributing.md for getting it running on your machine. If you have any trouble with that, please let me know and I'd be happy to help